.gitignore

```diff
@@ -43,3 +43,6 @@ compile_commands.json
 
 # Pyenv files
 .python-version
+
+# Prebuilt tools
+Prebuilt/
```

Comment:
Can we update the instructions to move this out-of-tree? That makes it easier to have multiple checkouts without needing to bootstrap (and then update) each one independently. An extra `..` for example, or `$HOME/` as the root for the artifacts, etc would work. This will also make it easier when packaging this for vcpkg, where we'll be referencing an externally-installed version.

Reply:
Sure. We can let user to choose the tool download path and export the path to cmake.